tests/test_search.py

```diff
@@ -2,15 +2,21 @@
 
 from lxml import etree, isoschematron
 
+from search import search
 from tests.schema_utils import get_schema_file_path
 
 
-def test_empty_response(client) -> None:
+def test_empty_response(client, mocker):
+    es_host = 'http://libero-test-elasticsearch'
+    search.ELASTICSEARCH_HOSTS = [es_host]
```

Comment:
stubs a setting for test but never resets it to the existing value; being this a fake URL, do we need to stub it at all? A real request to this would fail, but if the tests pass it means there are no real requests being made by the class constructor this is passed to. Undecided.

Reply:
I can remove these but, should unit tests rely on application settings or should they be able to run regardless of what the current settings are?

Reply:
Ok. I'm going to define default test settings that will be reset for each test. Any test that requires a setting with a different value can override the setting in the test.